mailing_list/views.py

```
@@ -83,6 +83,43 @@ def _is_rate_limited(request) -> bool:
+                    defaults={"email": email, "status": SubscriptionStatus.PENDING},
+                )
+            succeeded.append(lid)
+        except IntegrityError:
+            pass
+
+    if not succeeded:
+        return [], None
+
+    try:
+        _send_confirmation_email(request, email, user.pk, succeeded)
+    except Exception as exc:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)